var/spack/repos/builtin/packages/py-minkowskiengine/package.py

```diff
@@ -19,6 +19,9 @@ class PyMinkowskiengine(PythonPackage, CudaPackage):
 
     depends_on("py-setuptools", type="build")
     depends_on("py-pybind11", type="link")
+    # in newer pip versions --install-option does not exist
+    depends_on("py-pip@:23.0", type="build")
+    depends_on("openblas")
```

Comment:
Does any blas work?

Reply:
According to the [installation guide](https://nvidia.github.io/MinkowskiEngine/overview.html#installation), it should work with different blas (namely openblas, atlas, mkl, blas). I chose `openblas` because this is explicitly mentioned to install.

I now tried to swap it out with `blas` (the virtual package) but it fails with the error from above again (it was using `intel-oneapi-mkl`).
I manually tested other blas packages, but `intel-mkl` also give the error from above and `atlas` and `cblas` did not even build.

Reply:
Might have to explicitly set `--blas_include_dirs` and `--blas` in `install_options`. Would have to grep to find possible options for the latter.

Reply:
I did not manage to make it work with the other blas and most of the blas options available in spack are either not supported or don't build anyway.

Reply:
Can you add a comment saying that other BLAS should theoretically work but this exercise is left to the reader? Then if someone really cares they can make it work themselves.

Reply:
Done